kuma/wiki/tests/test_templates.py

```diff
@@ -680,6 +680,7 @@ def test_tag_list(self):
         doc = pq(response.content)
         eq_(1, len(doc('#document-list ul.document-list li')))
 
+    @pytest.mark.xfail(reason='Requires migrations')
```

Comment:
Test don't run in SCL3, they run in TravisCI, Vagrant, and now Docker.  But I'm not sure if it matters - there are different code paths for running tests with and without migrations, and this is the first time they have ever been run without migrations, which appears to make a difference for this table definition.  That's why it hasn't been discovered before.